### ISSUE TITLE
[Feature]: Assign tasks to people

### ISSUE THREAD

**Issue by pgebert** [mgmeyers/obsidian-kanban#770]
### Goal or desired outcome of this feature

We use a kanban board to coordinate tasks within a group of people. It would be very useful to assign a task to a person or multiple people by mentioning them in the task title to keep track who is working on which task.

### Describe the feature

If you type @ + name (e.g. `@John Doe`) in the task title it gets rendered as visible item like tags. The assigned persons should be rendered above or below the tags in a dedicated section of the task card.

To not infer with the @ + date, you could check with a regex whether the @ character is followed by a date or a name.

### Can you think of any alternatives or work-arounds?

(1) Currently we assign people by using tags. But that's not the actual purpose of tags and mixing up names and real tags gets quite chaotic.

(2) [obsidian_supercharged_links](https://github.com/mdelobelle/obsidian_supercharged_links): Plugin to assign custom styles to links and triggers. You could build the described features with it and another plugin. But that's quite complicated for a basic feature of a kanban board. And it does not get synced accross users.

(3) (obsidian-simple-mention )[https://github.com/der-tobi/obsidian-simple-mention]: A simpler plugin to achieve the functionality, but the styling is horrible in the tasks titles. And the people tags are not in a dedicated section in the bottom of the task card. 

### Screenshots, mockups, or videos

The feature could look like this:

![image](https://user-images.githubusercontent.com/6838540/233847680-58a31fe8-dd45-4d69-b346-e39c2505e5d7.png)

**Comment by mbbroberg** [comment]
I'm not a maintainer here, but optimizing this plugin for multiple users feels like a much different scope than it is for a personal setup. As a user, I'd prefer this kind of functionality separates into another plugin to keep this one simpler to maintain. I want to +1 that Super Charged Links plugin can work for this kind of functionality. Just a thought ✌️✌️

**Comment by pgebert** [comment]
I totally agree with you that a full multi user board in the manner of e.g. Jira would exceed the scope of this project. Neverthless, I think that basic name tags could be a useful feature and it shouldn't be too complicated to implement. 

If the maintainers of this project  consider this as useful I may could open a PR for it.

**Comment by darthwalsh** [comment]
Thanks for describing this issue!

If multi-users is out of scope for this plugin, maybe I should use another tool like Asana/Trello/Jira to manage our family household tasks... but so far it seems OK to use tags for assignees. (I'm not sure there's an objective "actual purpose" to tags, -- but if it's getting noisy would it help to namespace the tag like `#assign/carl`.)

It would be nice if the Obsidian Kanban board could display a custom icon based on the tag, like how Jira shows my avatar profile picture?

Right now I have a simple "dashboard" using the Obsidian Tasks plugin to query tasks that have my tag `#carl`, which has been working for me:

````
```tasks
path includes FamilyBoard
tags include #carl
not done
```
````

**Comment by LotharieSlayer** [comment]
@pgebert Since I don't like the tag system to mention people and even if I think it's a bit out of scope too, you can find a compatibility implementation of [obsidian-simple-mention](https://github.com/der-tobi/obsidian-simple-mention) on my [fork here](https://github.com/LotharieSlayer/obsidian-kanban) with [5f55215](https://github.com/LotharieSlayer/obsidian-kanban/commit/5f552152a6880d7ccad1a52fe4e761c715523dfc). I can PR it here if maintainers are interested tho.
